flink-table/flink-sql-jdbc-driver/src/main/java/org/apache/flink/table/jdbc/FlinkConnection.java

```diff
@@ -18,34 +18,67 @@
 
 package org.apache.flink.table.jdbc;
 
+import org.apache.flink.annotation.VisibleForTesting;
+import org.apache.flink.configuration.Configuration;
+import org.apache.flink.table.client.gateway.Executor;
+import org.apache.flink.table.client.gateway.StatementResult;
+import org.apache.flink.table.gateway.service.context.DefaultContext;
+import org.apache.flink.table.jdbc.utils.DriverUtils;
+
 import java.sql.DatabaseMetaData;
 import java.sql.SQLClientInfoException;
 import java.sql.SQLException;
 import java.sql.SQLFeatureNotSupportedException;
 import java.sql.Statement;
+import java.util.Collections;
 import java.util.Properties;
+import java.util.UUID;
 
 /** Connection to flink sql gateway for jdbc driver. */
 public class FlinkConnection extends BaseConnection {
-    private final DriverUri driverUri;
+    private final Executor executor;
+    private volatile boolean closed = false;
 
     public FlinkConnection(DriverUri driverUri) {
-        this.driverUri = driverUri;
+        // TODO Support default context from map to get gid of flink core for jdbc driver
```

Comment:
It would be great if you also add the JIra link in the comment.

Reply:
Thanks @libenchao , added the jira link